ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ccj/CountyCourtJudgmentMapper.java

```diff
@@ -44,29 +48,34 @@ public CCDCountyCourtJudgment to(CountyCourtJudgment countyCourtJudgment) {
             builder.ccjType(CCDCountyCourtJudgmentType.valueOf(ccjType.name()))
         );
 
+        Optional.ofNullable(claim.getCountyCourtJudgmentRequestedAt()).ifPresent(builder::requestedDate);
```

Comment:
no need to check optional if present as we are not processing on date, as well as by default it is already defaulted to null. so no harm in setting the date directly in builder

Reply:
Defensive programming is always good. But I have changed as asked.